demos/sahi/README.md

```diff
@@ -0,0 +1,16 @@
+# SAHI: Slicing Aided Hyper Inference Demo
+
+An example of how to use Norfair along with [SAHI: Slicing Aided Hyper Inference](https://github.com/obss/sahi )
```

Comment:
Remove space before )

Reply:
Mention that we are using a YOLOv5 detector

Reply:
Fixed! Thanks for catching this